public/css/jqueryfullPage.css

```diff
@@ -184,15 +184,16 @@ html.fp-enabled,
     transition: all 0.1s ease-in-out;
 }
 #fp-nav ul li:hover a span,
-.fp-slidesNav ul li:hover a span{
+.fp-slidesNav ul li:hover a span{
```

Comment:
It's the fullpage library css, i just had to change the colour of the navigation buttons but I agree!